contents/blog/posthog-changelog.md

```diff
@@ -19,10 +19,41 @@ Interested in what new features, fixes and updates have just launched on PostHog
 
 > Some new features you read about below may still be in beta, behind a feature flag, or only available to paying users. Want to see which betas are currently available? [Check the roadmap](/roadmap)!
 
+## March 10, 2023 
+
+### Dashboard templates
+![dashboard templates](../images/blog/array/dashboard-template-wizard.png)
+
+We want to make it easier and faster for users to find useful information in PostHog — and dashboards are a key part of that because they’re often one of the first things that users build. So, we’ve added a new selection of dashboard templates, as well as a new wizard that’s a little easier on the eye. 
+
+We’ve added a few simple templates to start with, for getting insights into areas such as online advertising, website traffic and user research. Got other ideas? Let us know in [the PostHog Slack](/slack)!
+
+### Hedgehog Mode collisions
+![hedgehog mode collisions](../images/blog/array/when-hogs-collide.gif)
+
+Hedgehog mode has had a frankly startling amount of improvements in recent weeks, but the latest is one of the most fun yet. We’ve now added collision detection, so that you can make Max ([that’s his name](/handbook/company/team) jump around all over the PostHog UI. 
+
+Is this important? Is it our defining feature? Is it the foundation for even crazier projects to come at our next hackathon? The answer to all of these questions is, yes, probably - and you can turn on Hedgehog Mode via the in-app drop down menu. 
+
+### Feature flags for Flutter
+We always love adding user contributions into PostHog, especially when they have such a storied past as this one. In June 2021, we [asked for contributions](https://github.com/PostHog/posthog-flutter/issues/14) to our Flutter library, to extend feature flag support there. In August 2022, [Etienne Théodore](https://github.com/Kiruel) stepped up to the plate and took on the task. We were admittedly slow getting this PR approved, but can now happily say that our Flutter library includes feature flag support. 
```

Comment:
We should update docs for this: https://posthog.com/docs/sdks/flutter

Reply:
Tagging @pjhul for this, as it's way outside my area of knowledge